packages/cashc/test/compiler/InvalidParameterTypeError/locking_bytecode_p2pkh_bytes32.cash

```diff
@@ -0,0 +1,8 @@
+pragma cashscript ^0.7.0;
+
+contract P2PKH(bytes20 pkh) {
+    function spend(pubkey pk, sig s) {
+        require(checkSig(s, pk));
+        require(tx.outputs[0].lockingBytecode == new LockingBytecodeP2PKH(sha256(pkh)));
+    }
+}
```

Comment:
FWIW I can't see what the invalid part here is. I'm 99% sure that's due to my lack of familiarity. Just mentioning it in case it's something else.

Reply:
`new LockingBytecodeP2PKH(bytes20 pkh)` expects a `bytes20` type to be passed, but `sha256` returns `bytes32` so those types are incompatible.